.github/pull-request-tasks.yml

```diff
@@ -0,0 +1,24 @@
+name: Swift Integration Tests
+
+on:
+  push:
+    branches:
+      - master
+  pull-request:
+
+jobs:
+  test:
+    name: Test
+    runs-on: macos-latest
+    strategy:
+      matrix:
+        destination: ['platform=iOS Simulator,OS=latest,name=iPhone 12 Pro Max']
```

Comment:
do we want to run tests on multiple OS versions and devices?

Reply:
by adding other entries to this array we get jobs run in parallel - i.e. if we added `platform=iOS Simulator,OS=latest,name=iPhone 11 Pro` to this another job would spin up and test that simulated device at the same time.

Not sure if that's a good approach, for now tests are only running on a single simulated device.

Reply:
One device is OK for now.